config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: cass-operator
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: cass-operator-
+
+# Labels to add to all resources and selectors.
+#commonLabels:
+#  someName: someValue
+
+bases:
```

Comment:
I believe `bases` is deprecated and should be replaced with `resources`.

Do we want to include certmanager by default?

Reply:
bases is the naming used by the newest version of operator-sdk

Reply:
If it is deprecated though wouldn't it be better to update?

Reply:
It will break users who use kustomize in kubectl <1.20. It will change behavior to those users.

Reply:
Ah, I always had problems with the version of kustomize kubectl so I installed it separately.

For reference https://github.com/kubernetes/kubectl/issues/891#issuecomment-731238572.